_pkgdown.yml

```diff
@@ -14,26 +14,42 @@ home:
     href: http://cvpia.scienceintegrationteam.com/
     
 reference:
-- title: EML Element Functions
+- title: Create EML Element Functions
+  desc: Functions for creating metadata sections.
+  contents:
+  - create_abstract
```

Comment:
plz, confirm that all functions that need to be here are

Reply:
Confirmed.